tests/cli/formatters/test_storage_formatters.py

```diff
@@ -415,11 +415,11 @@ def test_long_formatter(self) -> None:
 
         formatter = LongFilesFormatter(human_readable=True, color=False)
         assert list(formatter(self.files_and_folders)) == [
-            "-r    2.0K 2018-01-01 03:00:00 File1",
-            "-r    1.0K 2018-10-10 13:10:10 File2",
-            "-r 1000.0K 2019-02-02 05:02:02 File3 with space",
-            "dm       0 2017-03-03 06:03:03 Folder1",
-            "dm       0 2017-03-03 06:03:02 1Folder with space",
+            "-r    2.0 Ki 2018-01-01 03:00:00 File1",
```

Comment:
Why does it show "Ki" and not "KiB" as for disks formatter?

Reply:
It was striped in formatter (here is how it looks like on master brunch):
https://github.com/neuromation/platform-client-python/blob/18ecc3cf577a16c44df41fd24f14335e7cf10dc0/neuromation/cli/formatters/storage.py#L477-L478
Now I fixed that.